make/karma-targets.mk

```diff
@@ -3,22 +3,27 @@
 # Copyright (c) 2015 Cyan, Inc. All rights reserved.
 #
 
+TESTING_FRAMEWORK ?= jasmine
+
 .PHONY: \
 	karma-coverage \
 	karma-test \
 	karma-watch
 
 karma-coverage: export JASMINE=1
```

Comment:
Seems like we should change this as well. I'll add an issue to track it. Now that we support "mocha" it should probably be a more generic env variable.